tools/init.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+psql <<EOF
+CREATE EXTENSION plpython3u;
+CREATE EXTENSION rikai;
+
+INSERT INTO ml.models (name, flavor, model_type)
```

Comment:
Is the plan to add a `CREATE MODEL` syntax like the other pg ml project has?

Reply:
It might not be.  `CREATE MODEL` requires changes in the parser. I have not figured out how to do it in postgres yet.